packages/shared/src/components/notifications/InAppNotification.tsx

```diff
@@ -0,0 +1,124 @@
+import React, { ReactElement, useContext, useEffect, useState } from 'react';
+import { useRouter } from 'next/router';
+import { useQuery, useQueryClient } from 'react-query';
+import classNames from 'classnames';
+import {
+  InAppNotification,
+  IN_APP_NOTIFICATION_KEY,
+  useInAppNotification,
+} from '../../hooks/useInAppNotification';
+import classed from '../../lib/classed';
+import { isTouchDevice } from '../../lib/tooltip';
+import CloseButton from '../CloseButton';
+import { InAppNotificationItem } from './InAppNotificationItem';
+import styles from './InAppNotification.module.css';
+import { useAnalyticsContext } from '../../contexts/AnalyticsContext';
+import { AnalyticsEvent, Origin } from '../../lib/analytics';
+import { NotificationType } from '../../graphql/notifications';
+import FeaturesContext from '../../contexts/FeaturesContext';
+import { InAppNotificationPosition } from '../../lib/featureValues';
+
+const Container = classed(
+  'div',
+  classNames(
+    styles.inAppNotificationContainer,
+    'animate-bounce',
+    'fixed right-10 bg-theme-bg-notification border border-theme-active rounded-16 in-app-notification slide-in z-[100] w-[22.5rem] h-22',
```

Comment:
Nothing urgent, but thinking about this, maybe we should specify a z-index sheet at one stage.
No action required I'll note it somewhere.